src/ReadLoader.hpp

```diff
@@ -30,6 +32,7 @@ class shasta::ReadLoader :
         size_t threadCount,
         const string& dataNamePrefix,
         size_t pageSize,
+        const PalindromicReadOptions& palindromicReadOptions,
```

Comment:
I would prefer to pass to the `ReadLoader` only the three parameters it needs, rather than the entire `PalindromicReadOptions`. Passing the entire object makes it hard for the reader to keep track of where options are used.